docs/Tutorial.md

```diff
@@ -32,7 +32,7 @@ class HelloWorldApp extends Component {
 AppRegistry.registerComponent('HelloWorldApp', () => HelloWorldApp);
 ```
 
-If you are feeling curious, you can play around with sample code directly in the web simulators. You can also paste it into your `index.ios.js` or `index.android.js` file to create a real app on your local machine.
+If you are feeling curious, you can play around with sample code directly in the web simulators. If you are using AwesomeProject from the Getting Started tutorial, you can also paste the sample code into your `index.ios.js` or `index.android.js` file to create a real app on your local machine. Change the appKey from `'HelloWorldApp'` to `'AwesomeProject'` in line 32.
```

Comment:
This adds complexity for some readers - "if you are using" - those who don't will need to parse and skip this sentence. Also the line number 32 could change and no one will remember to update this Tutorial.

IMHO this would be better to document directly in the Getting Started tutorial because it will be relevant to the people reading that.